app/build.gradle

```diff
@@ -203,3 +202,18 @@ tasks.withType(org.jetbrains.kotlin.gradle.internal.KaptWithoutKotlincTask) {
         "--add-opens", "jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED",
         "--add-opens", "jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED"])
 }
+
+tasks.register("newLintBaseline") {
```

Comment:
What is "trigger" to recreate that?

I never  had to do it but for the last PR I did?

Reply:
The lint baseline records lint errors that we're currently ignoring. It does that by filename and line number.

If you move code around, or add new code to files that have lint errors then it can mean that the information in the baseline is obsolete.

E.g., in the errorphant PR the lint baseline had entries for the files under the old names (because the lint was complaining about the complex vector path). When you renamed the files the entries in the baseline file became obsolete, and lint started complaining about the new files.

This is a convenience task so that you can (a) run it from the command line (so it's one commend instead of two), and (b) add it as a task in Android studio so it's a single click, like so:

![image](https://github.com/tuskyapp/Tusky/assets/773100/686debe3-d1e4-4510-93ef-6ac0f8292661)